executor/aggfuncs/aggfunc_test.go

```diff
@@ -242,10 +240,10 @@ func distinctUpdateMemDeltaGens(srcChk *chunk.Chunk, dataType *types.FieldType)
 			val = strconv.FormatInt(row.GetInt64(0), 10)
 			memDelta = aggfuncs.DefInt64Size
 		case mysql.TypeJSON:
-			json := row.GetJSON(0)
+			jsonVal := row.GetJSON(0)
 			bytes := make([]byte, 0)
-			bytes = append(bytes, json.TypeCode)
-			bytes = append(bytes, json.Value...)
+			bytes = append(bytes, jsonVal.TypeCode)
```

Comment:
`json.TypeCode` has had calculated in `AllocPartialResult`, we don't need calculate it anymore.

Reply:
`distinctUpdateMemDeltaGens` do not called in my test cases

Reply:
my mistake.